doc/installing-a-development-environment.md

```diff
@@ -1,5 +1,14 @@
 ## Installing
 
+### Introduction to Installation
+
+The development environment setup requires installation of following softwares:
+
+1. Ruby - Needs to be installed in order to install and use Jekyll. Jekyll helps in tranforming plain texts into static websites.
+1. Node.js - Needs to be installed in order to install all the required development and javascript dependencies.
```

Comment:
How about: "Needed by the build tools."

Reply:
@dblotsky  I feel its too short and abstract. Any elaborations?
@janpio Your thoughts?

Reply:
@janpio @dblotsky  Should we elaborate it bit more or just say "Needed by the build tools"? If its clarified, we can close this PR